src/app/projects/states/portfolio/directives/portfolio-grade-select-step/portfolio-grade-select-step.component.ts

```diff
@@ -0,0 +1,65 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {Injector} from '@angular/core';
+import angular from 'angular';
+import {ProjectService} from 'src/app/api/services/project.service';
+import {GradeService} from 'src/app/common/services/grade.service';
+import {ViewEncapsulation} from '@angular/core';
+
+@Component({
+  selector: 'portfolio-grade-select-step',
+  templateUrl: 'portfolio-grade-select-step.component.html',
+  styleUrls: ['portfolio-grade-select-step.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+})
+export class PortfolioGradeSelectStepComponent implements OnInit {
+  @Input() project: any;
+  @Input() unit: any;
```

Comment:
Add typing to the inputs?
```suggestion
  @Input() project: Project;
  @Input() unit: Unit;
```